ourrevolution/settings.py

```diff
@@ -77,16 +77,17 @@
     #
     'wagtail.contrib.wagtailfrontendcache',
     'wagtail.contrib.wagtailroutablepage',
-    
+
     # Group Goodies
     'phonenumber_field',
     'multiselectfield',
     'django_countries',
     'recurrence',
     'address',
-    
     'bootstrap3',
     'crispy_forms',
+    'ckeditor',
```

Comment:
Why do we need a new editor?

Reply:
This is for rich text, WYSIWYG for staff to add more than just text. Mostly this is for list functionality because this will be used to generate the candidate write-up.